LICENSE

```diff
@@ -1,7 +1,6 @@
-
                                  Apache License
                            Version 2.0, January 2004
-                        https://www.apache.org/licenses/
+                        http://www.apache.org/licenses/
```

Comment:
The HTTPS form is [not my work][1].  I think consistency with project-template is more important than keeping an HTTPS here, especially as the [canonical source for this license uses HTTP][2].

[1]: https://github.com/docker/opensource/commit/fcff91e6eb3f6798e538bdc8f9e34a78f457163b#diff-86dbfda638ac132a424a7a1dec5e619aR4
[2]: http://www.apache.org/licenses/LICENSE-2.0.txt